cumulusci.yml

```diff
@@ -480,7 +480,7 @@ tasks:
                 - path: src/objectTranslations/DataImport__c-**.objectTranslation
                   xpath: //ns:fields/ns:help[../ns:name='Donation_Name__c']
                 - path: src/objectTranslations/DataImport__c-**.objectTranslation
-                  xpath: //ns:fields/ns:help[../ns:name='Donation_Record_Type_Name__c__c']
```

Comment:
I noticed that this had an accidental `__c__c` in the name match. I must have left that in there from some earlier testing when I built the task. Removing since it should not be there.